src/GafferScene/InteractiveRender.cpp

```diff
@@ -469,3 +472,18 @@ Gaffer::ValuePlug::CachePolicy InteractiveRender::computeCachePolicy( const Gaff
 
 	return ComputeNode::computeCachePolicy( output );
 }
+
+bool InteractiveRender::acceptsInput( const Gaffer::Plug *plug, const Gaffer::Plug *inputPlug ) const
+{
+	if( plug == rendererPlug() || plug == statePlug() )
+	{
+		// We can't accept computed plugs as we may need to dirty our messages plug as
+		// result. It is forbidden to dirtying additional plugs during dirty propagation.
```

Comment:
to dirtying additional plugs -> to modify the node graph

Reply:
Fixed in https://github.com/GafferHQ/gaffer/commit/a29fa8721bedf84c9282a780367b63e47ca0a2b1